modules/serverless-open-telemetry.adoc

```diff
@@ -0,0 +1,149 @@
+
+.Procedure
+
+. Create an `OpenTelemetryCollector` custom resource (CR):
++
+.Example OpenTelemetryCollector CR
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: tracing-system
+spec:
+  mode: deployment
+  config: |
+    receivers:
+      zipkin:
+    processors:
+    exporters:
+      jaeger:
+        endpoint: jaeger-all-in-one-inmemory-collector-headless.tracing-system.svc:14250
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+      logging:
+    service:
+      pipelines:
+        traces:
+          receivers: [zipkin]
+          processors: []
+          exporters: [jaeger, logging]
+----
+
+. Verify that you have two pods running in the namespace where {DTProductName} is installed:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                          READY   STATUS    RESTARTS   AGE
+cluster-collector-collector-85c766b5c-b5g99   1/1     Running   0          5m56s
+jaeger-all-in-one-inmemory-ccbc9df4b-ndkl5    2/2     Running   0          15m
+----
+
+. Verify that the following headless services have been created:
++
+[source,terminal]
+----
+$ oc get svc -n tracing-system | grep headless
+----
++
+.Example output
+[source,terminal]
+----
+cluster-collector-collector-headless            ClusterIP   None             <none>        9411/TCP                                 7m28s
+jaeger-all-in-one-inmemory-collector-headless   ClusterIP   None             <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   16m
+----
++
+These services are used in to configure Jaeger and Knative Serving. The name of the Jaeger service may vary.
+
+. Install the {ServerlessOperatorName} by following the "Installing the {ServerlessOperatorName}" documentation.
+
+. Install Knative Serving by creating the following `KnativeServing` CR:
++
+.Example KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+    name: knative-serving
+    namespace: knative-serving
+spec:
+  config:
+    tracing:
+      "backend": "zipkin"
+      "zipkin-endpoint": "http://cluster-collector-collector-headless.tracing-system.svc:9411/api/v2/spans"
+      "debug": "true"
+      sample-rate: "1" <1>
+----
+<1> The sample rate determines what percentage of traces are stored for visualization.
+
+. Create a Knative service:
++
+.Example service
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+spec:
+  template:
+    metadata:
+      labels:
+        app: helloworld-go
+      annotations:
+        autoscaling.knative.dev/minScale: "1"
+        autoscaling.knative.dev/target: "1"
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/helloworld:v1.2
```

Comment:
@mgencur is there another productized image for the upstream hellowrold-go?

Reply:
If it's just a generic service image we need we can use the same as the one here https://docs.openshift.com/container-platform/4.10/serverless/develop/serverless-applications.html i.e. `quay.io/openshift-knative/knative-eventing-sources-event-display:latest` maybe?

Reply:
I know about https://quay.io/repository/openshift-knative/helloworld-go but it's older and not updated regularly.

Reply:
Which do we want to go with? 🙂

Reply:
The rest of this PR now looks good. I'd probably go with quay.io/openshift-knative/helloworld:v1.2